website/docs/library/restable.md

```diff
@@ -1,25 +1,239 @@
+|    13 |                                            43.870 |                           22.135 |
+|    14 |                                             6.267 |                            3.162 |
+|    15 |                                             0.895 |                            0.452 |
+
+*: Within a given resolution, all pentagons have the same area.
+
+
+### Hexagon min and max areas
+
+The area of an H3 cell varies based on its position relative to the
+[icosahedron vertices](../core-library/overview.md).
+We compute the minimum and maximum values for the **hexagon** areas (excluding
+the pentagons) at each resolution, and show their ratio.
+
+
+|   Res |   Min <ins>Hexagon</ins> Area (km^2) |   Max <ins>Hexagon</ins> Area (km^2) |   Ratio (max/min) |
+|------:|-------------------------------------:|-------------------------------------:|------------------:|
+|     0 |                  4,106,166.334463915 |                  4,977,807.027442012 |          1.212276 |
+|     1 |                    447,684.201817940 |                    729,486.875275344 |          1.629468 |
+|     2 |                     56,786.622889474 |                    104,599.807218925 |          1.841980 |
+|     3 |                      7,725.505769639 |                     14,950.773301379 |          1.935248 |
+|     4 |                      1,084.005635363 |                      2,135.986983965 |          1.970457 |
+|     5 |                        153.766244448 |                        305.144308779 |          1.984469 |
+|     6 |                         21.910021013 |                         43.592111685 |          1.989597 |
+|     7 |                          3.126836030 |                          6.227445905 |          1.991613 |
+|     8 |                          0.446526174 |                          0.889635157 |          1.992347 |
+|     9 |                          0.063780227 |                          0.127090737 |          1.992635 |
+|    10 |                          0.009110981 |                          0.018155820 |          1.992740 |
+|    11 |                          0.001301542 |                          0.002593689 |          1.992782 |
+|    12 |                          0.000185933 |                          0.000370527 |          1.992797 |
+|    13 |                          0.000026562 |                          0.000052932 |          1.992802 |
+|    14 |                          0.000003795 |                          0.000007562 |          1.992805 |
+|    15 |                          0.000000542 |                          0.000001080 |          1.992805 |
+
+
+## Edge lengths
+
+:::caution
+Edge lengths are computed with a **spherical** model of the earth using the
+[authalic radius](https://en.wikipedia.org/wiki/Earth_radius#Authalic_radius)
+given by
+[WGS84](https://en.wikipedia.org/wiki/WGS84)/[EPSG:4326](https://epsg.io/4326).
+:::
+
+TODO
```

Comment:
Do we want to keep the TODOs here, or leave/comment the sections out? This seems a little odd in published docs.

Reply:
I'm only changing the 4.0 docs in this diff, which aren't the ones surfaced to users by default. In that sense, I'm considering the 4.0 docs a work in progress, so I'm not too bothered by obvious `TODO` sections that'll hopefully remind us to fill those out before finalizing the 4.0 docs.